server/webapp/WEB-INF/rails.new/app/assets/stylesheets/structure.scss

```diff
@@ -271,6 +271,10 @@ body[dir='ltr'] {
   padding: 10px;
 }
 
+#MB_content .modal-box-section {
+  padding: 0px 15px 0px 15px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0px 15px` instead of `0px 15px 0px 15px`<br>`0px` should be written without units as `0`